### ISSUE TITLE
Combat system is broken, not sure what's going on

### ISSUE THREAD

**Issue by Drigonis** [daleao/modular-overhaul#157]
sorry but i really can't provide much information here!! logs aren't telling me anything, it may be a mod conflict but probably not. i use animation cancelling all the time (i've bound it to the spacebar) and i usually animation cancel my weapons after hitting an enemy, it's just instinct now; makes it much faster. but when animation cancelling with MARGO's combat system, it just.. breaks. you can't swing your sword anymore. you can, however, hold down left-click, and walk sideways, and you'll move slightly downwards as you walk, because it still thinks you're swinging and moves you downward if that module is enabled. i'm not exactly sure how i usually fix this, but i think switching to another tool, and animation-cancelling it before it hits, is what fixes it.

**Comment by daleao** [comment]
I don't know what you mean by animation cancelling.
Please provide exact step-by-step instructions to reproduce your issue and please include a SMAPI log.

**Comment by Drigonis** [comment]
not sure how you've never managed to hear of it - animation cancelling is a set of keys you can press to "cancel" an animation which is used by speedrunners and such to be able to immediately start swinging a tool after it hits the ground, by cancelling the animation once it hits the ground and then swinging again. it makes the process much faster and is pretty commonly used nowadays. it can be done by pressing Right Shift + R + Delete, but usually people just assign those keys to a hotkey like the spacebar.
tldr; to reproduce this issue, swing a sword (in this case, i was using the Wooden Blade) and then press Right Shift + R + Delete, to cancel the animation. you'll be unable to swing your sword, but holding left-click and walking perpendicular to the direction you swung in will slowly drag you toward that direction, as if you're still being pulled by the swing's movement.

**Comment by daleao** [comment]
I understand what animation canceling is in general. But I'm not familiar with speedrunning hacks in SDV.
The logic of the combo framework depends on the complete execution of the animation. I.e., each subsequent attack triggers on completion of the previous animation. So I'm not surprised that bypassing it causes problems.
Best I can do is issue a warning to the documentation.
